### PR TITLE
Branch add sort property command

### LINKED ISSUE #17: As a user, I want to be able to sort all of the properties based on location
so that I can recommend suitable properties to buyers

### LINKED ISSUE #18: As a user, I want to be able to sort property listings based on the recency of property listings
so that I can update clients on newly listed properties

### PR DESCRIPTION
**Pull request by Cp-John**

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -204,9 +204,15 @@
     void updateFilteredAppointmentList(Predicate<Appointment> predicate);
 
     /**
-     * Sorts the the filtered appointment list by the given {@code comparator}.
+     * Sorts the the appointment list by the given {@code comparator}.
```
Comment (dvdweien) [inline src/main/java/seedu/address/model/Model.java]: shouldn't it sort the filtered list?

File: src/test/java/seedu/address/logic/commands/SortPropertyCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import seedu.address.model.property.Property;
+import seedu.address.testutil.SortPropertyDescriptorBuilder;
+
+
+public class SortPropertyCommandTest {
+
+    private Model model = new ModelManager(getTypicalPropertyBook(), new UserPrefs());
+
+    @Test
+    public void execute_filterByDeadlineInAscendingOrder_success() {
```
Comment (dvdweien) [inline src/test/java/seedu/address/logic/commands/SortPropertyCommandTest.java]: should it be filterby or sortby? same for the rest below

**Review by dvdweien — COMMENTED** [review]
LGTM, just some things that need clarification

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (Cp-John) [reply src/main/java/seedu/address/model/Model.java]: The method actually sorts the appointment list but the displayed filtered appointment list is sorted accordingly. Should I change back the javadoc comment?

File: src/test/java/seedu/address/logic/commands/SortPropertyCommandTest.java (hunk 2 above)
Comment (Cp-John) [reply src/test/java/seedu/address/logic/commands/SortPropertyCommandTest.java]: updated

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (dvdweien) [reply src/main/java/seedu/address/model/Model.java]: yeah, sure

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**